### PR TITLE
cleanups

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

**Comment by monstermunchkin** [comment]
Would you mind doing this kind of cleanup for other commands as well? The `Command`s in `api_cluster.go` for example could use some cleaning up as well.

**Comment by brauner** [comment]
> Would you mind doing this kind of cleanup for other commands as well? The Commands in api_cluster.go for example could use some cleaning up as well.

Yip, will do. I just did the ones I remembered right away. :)

File: lxd/images.go [hunk 1]
```diff
@@ -40,6 +40,54 @@ import (
...
+var imageCmd = Command{
+	name:         "images/{fingerprint}",
+	untrustedGet: true,
+	get:          imageGet,
+	put:          imagePut,
+	delete:       imageDelete,
+	patch:        imagePatch,
+}
+
+var imagesExportCmd = Command{
```
[the remark this answers is not in the document]
Comment (brauner) [reply lxd/images.go]: addressed

File: lxd/images.go [hunk 2]
```diff
@@ -40,6 +40,54 @@ import (
...
+	name:         "images/{fingerprint}/export",
+	untrustedGet: true,
+	get:          imageExport,
+}
+
+var imagesSecretCmd = Command{
```
[the remark this answers is not in the document]
Comment (brauner) [reply lxd/images.go]: addressed

File: lxd/images.go [hunk 3]
```diff
@@ -40,6 +40,54 @@ import (
...
+	name: "images/{fingerprint}/secret",
+	post: imageSecret,
+}
+
+var imagesRefreshCmd = Command{
```
[the remark this answers is not in the document]
Comment (brauner) [reply lxd/images.go]: addressed

**Comment by brauner** [comment]
@monstermunchkin

**Comment by monstermunchkin** [comment]
Looks good

**Pull request merged**